pkg/deploy/taskdir/definitions/schema_0_3.json

```
@@ -390,14 +390,37 @@
           }
         },
         "resources": {
-          "description": "[Experimental] A map of resources that can be accessed from your task. Keys determine how you refer to the resource; values are existing resource slugs in your Airplane team.",
-          "type": "object",
-          "patternProperties": {
-            "^[a-z0-9_]{1,50}$": {
-              "type": "string",
-              "pattern": "^[a-z0-9_]{1,50}$"
+          "description": "[Experimental] A map of resources that can be accessed from your task. Keys are aliases, which determine how you refer to the resource in your task; values are existing resource slugs in your Airplane team. Can also be a list of resource slugs (where the alias is the slug) or a list of objects with a single alias -> slug mapping.",
```

IMO it's more clear to explain w/out aliases first:

```suggestion
          "description": "A list of resources to make available to your task. Resources are identified by slug, but can be mapped to an alias to configure how the task references the resource. If using aliases, resources are expressed as a map of alias to slug.",
```